services/motion/builtin/builtin.go

```diff
@@ -668,3 +669,73 @@ func (ms *builtIn) planMoveOnMap(
 	steps, err := plan.GetFrameSteps(f.Name())
 	return steps, kb, err
 }
+
+// ErrorState will provide a pose P that is the kinematic base's offset from where it would be if it were to follow a path perfectly.
+// This pose P is such that spatialmath.Compose(nominalPose, P) will yield the actual pose seen by the localizer.
+func ErrorState(
```

Comment:
This is unnecessary at this point correct?

Reply:
Ack. I swear I deleted this. Apparently not.